modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_contributor_builder.scss

```diff
@@ -35,7 +35,13 @@
 				transition: all 0.5s ease;
 
 				.content-wrapper {
-					padding: 40px 24px;
+					padding: 18px 24px;
```

Comment:
In the figma I see a padding top of 24px. Maybe you can try to use a class for applying the padding and remove this css line.